kustomize/README.md

```diff
@@ -0,0 +1,228 @@
+## INSTALLING TERRASCAN IN A KUBERNETES CLUSTER USING KUSTOMIZE
+
+This guide will help you install terrascan server inside your kubernetes cluster, as per your specific use case.
+We have covered four use cases in the sections below:
+
+  ##### Deploying Terrascan Server
+  Terrascan operating in http server mode.
+
+  #### Deploying Terrascan Server in TLS Mode
+  Terrascan operating in https server mode. This deployment is also a foundation for the terrascan webhook setup.
+
+  #### Deploying Terrascan Server for Remote Repository Scan
```

Comment:
Can we try to indicate the user that in this section we would be authentication with ssh keys for scanning remote repositories?